processor/tailsamplingprocessor/README.md

```diff
@@ -67,7 +72,44 @@ processors:
             name: test-policy-7,
             type: rate_limiting,
             rate_limiting: {spans_per_second: 35}
-         }
+         },
+        {
+          name: composite-policy-1,
+          type: composite,
+          composite:
+            {
+              max_total_spans_per_second: 1000,
+              policy_order: [test-composite-policy-1, test-composite-policy-2, test-composite-policy-3],
+              composite_sub_policy:
+                [
+                  {
+                    name: test-composite-policy-1,
+                    type: numeric_attribute,
+                    numeric_attribute: {key: key1, min_value: 50, max_value: 100}
+                  },
+                  {
+                    name: test-composite-policy-2,
+                    type: string_attribute,
+                    string_attribute: {key: key2, values: [value1, value2]}
+                  },
+                  {
+                    name: test-composite-policy-3,
+                    type: always_sample
+                  }
+                ],
+              rate_allocation:
+                [
+                  {
+                    policy: test-composite-policy-1,
+                    percent: 50
+                  },
+                  {
+                    policy: test-composite-policy-2,
+                    percent: 25
+                  }
```

Comment:
I'm not sure I'm a fan of this. Implicitly, this config means that `test-composite-policy-3` has 25% of the rate. Someone looking at this config a few months after the initial configuration might be confused, as the missing entry can be interpreted as 0%.

I think I would rather fail at boot when the percentages don't add up to 100%.

Reply:
This doesn't have to add up to 100%. For example, lets say in the decision_wait window we want to capture 100% of all traces having errors. In case we still have capacity we can fill it with the traces matching other sub-policies as defined in policy order. This will help in filling our bucket with traces that matter the most. In the below example we want to capture 100% of all errors and just 1% of successful traces. 
{
          name: composite-policy-1,
          type: composite,
          composite:
            {
              max_total_spans_per_second: 1000,
              policy_order: [ error_policy, good_policy ],
              composite_sub_policy:
                [
                  {
                    name: error_policy,
                    type: numeric_attribute,
                    numeric_attribute: {key: http.status_code, min_value: 201, max_value: 500}
                  },
                  {
                    name: good_policy,
                    type: numeric_attribute,
                    numeric_attribute: {key: http.status_code, min_value: 0, max_value: 200}
                  }
                ],
              rate_allocation:
                [
                  {
                    policy: error_policy,
                    percent: 100
                  },
                  {
                    policy: good_policy,
                    percent: 1
                  }
                ]
            }
        },